espnet2/asr/frontend/s3prl.py

```diff
@@ -55,6 +56,13 @@ def __init__(
             "HubertModel",
         ]:
             upstream.model.encoder.layerdrop = 0.0
+            if getattr(self.args, "add_adapters", False):
```

Comment:
where is `self.args` from?

Reply:
I don't think this add_adapters is unified for all s3prl upstream. It is only for HuBERT and Wav2Vec.